### PR TITLE
Waf 396 filters

### PR DESCRIPTION
**Pull request by jerseyAdmin**
Work for all filters changes, ticket [WAF-396](https://amersports.atlassian.net/browse/WAF-396)

### CODE REVIEW

File: src/containers/App.tsx [hunk 1]
```diff
@@ -117,9 +109,6 @@ export const App: FunctionComponent = () => {
                         screenSize={screenSize}
                       />
                     </div>
-                    {displayByFranchise && (
```
Comment (chicharr) [inline src/containers/App.tsx:120]: isn't this needed anymore?

File: src/components/SortDropdown/SortDropdown.tsx [hunk 2]
```diff
@@ -22,32 +21,41 @@ export interface SortDropdownProps {
   mobile?: boolean;
 }
 
+export const toggleFilters = (event: Omit<MouseEvent, "currentTarget"> & {
```
Comment (chicharr) [inline src/components/SortDropdown/SortDropdown.tsx:24]: I understand this is toggling the filters & sort modal. I would move it to a different file, and not make it part of the SortDropdown

File: src/components/InputButtonGroup/InputButtonGroup.tsx [hunk 3]
```diff
@@ -123,25 +125,21 @@ export const InputButtonGroup: FunctionComponent<InputButtonGroupProps> = ({
...
-                <span className="font-button-2 text-[12px]">
-                  {translation.InputButtonGroup.showmore}
-                </span>
-              </button>
-            </div>
+          {attribute === 'price' && (
+            <LabelledInput
+              name={`range-radio-${attribute}`}
+              attribute={attribute}
+              label={'Range price'}
```
Comment (chicharr) [inline src/components/InputButtonGroup/InputButtonGroup.tsx:132]: shouldn't the label come from the translations file?

File: src/components/InputButtonGroup/InputButtonGroup.tsx [hunk 4]
```diff
@@ -123,25 +125,21 @@ export const InputButtonGroup: FunctionComponent<InputButtonGroupProps> = ({
...
+              checked={false}
+              value={'125.0-555.0'}
```
Comment (chicharr) [inline src/components/InputButtonGroup/InputButtonGroup.tsx:134]: what is this hardcoded value doing?

File: src/containers/App.tsx (hunk 1 above)
Comment (jerseyAdmin) [reply src/containers/App.tsx:120]: No, Franchise was moved into selected filters ccomponent.

**Pull request merged**